service/src/main/java/bio/terra/catalog/config/RawlsConfiguration.java

```diff
@@ -0,0 +1,6 @@
+package bio.terra.catalog.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "catalog.rawls")
+public record RawlsConfiguration(String basePath, String resourceId) {}
```

Comment:
I don't think we need `resourceId` for this config

Reply:
Whoops, a bit of copy-paste from the other configurations.